lib/graphql/types/int.rb

```diff
@@ -10,7 +10,7 @@ class Int < GraphQL::Schema::Scalar
       MAX = (2**31) - 1
 
       def self.coerce_input(value, _ctx)
-        value.is_a?(Numeric) ? value.to_i : nil
+        value.is_a?(Integer) ? value.to_i : nil
```

Comment:
Do we still need to call `.to_i` in the case that it's an Integer? Or could it be: 

```suggestion
        value.is_a?(Integer) ? value : nil
```